guides/common/assembly_configuring-and-setting-up-remote-jobs.adoc

```diff
@@ -1,6 +1,6 @@
 include::modules/con_configuring-and-setting-up-remote-jobs.adoc[]
 
-include::modules/con_about-running-jobs-on-hosts.adoc[leveloffset=+1]
+include::modules/con_remote-execution-in-foreman.adoc[leveloffset=+1]
```

Comment:
@maximiliankolb suggested to use upstream terminology rather than attribute keys in file names, hence the difference between file name and ID here. I'll keep this comment thread open in case you want to discuss it (I think I'm fine either way).

Reply:
@maximiliankolb This file is also part of the downstream. Using `foreman` in the filename is new to me. Can you explain in short?

Reply:
Because file names are AFAIK never shown/referenced in built docs that we offer to customers, I think it's a good compromise/convention to stick to upstream terminology. This clarifies things a lot if we go with "foreman-server", "smart-proxy", and "smart-proxy-server" everywhere instead of "project" and "proxy". I particularly dislike "proxy" because some people think "HTTP proxy".

This issue/comment is not blocking your PR. I'd also be happy to give it a try after you have merged this. I consider this a small QoL change for maintainers :)